contrib/pylightning/lightning/plugin.py

```diff
@@ -39,13 +113,15 @@ def __init__(self, stdout=None, stdin=None, autopatch=True):
         if os.getenv('LIGHTNINGD_PLUGIN') and autopatch:
             monkey_patch(self, stdout=True, stderr=True)
 
-        self.add_method("getmanifest", self._getmanifest)
+        self.add_method("getmanifest", self._getmanifest, sync=True)
```

Comment:
I don't have an whimsical ideas, but if `async` is taken, how about `is_async` ?

Reply:
Here's a few variants that I thought of that might also work: `direct=False`, `background=True`, `_async=True`.

I tend to like `background` since it sort of describes that we want the thing to continue in the background.

Reply:
+1 for `background`. i also like `wait=False`